modules/bullet/test/bullet_test.exs

```diff
@@ -0,0 +1,30 @@
+defmodule BulletTest do
+  use ExUnit.Case
+
```

Comment:
I started writing a comment about just invoking the bullet's tick/0 directly rather than sleeping, but then I realized that it is also good to test the timer logic in the servers, which is non trivial.